opte/src/api/mac.rs

```diff
@@ -94,4 +94,8 @@ impl MacAddr {
     pub fn bytes(&self) -> [u8; 6] {
         self.inner
     }
+
+    pub const fn from_const(bytes: [u8; 6]) -> Self {
```

Comment:
Is this to have a `const` version of `impl From<[u8; 6]>`?

Reply:
Yea, it allowed me to simplify test code (see the `int_test::GW_MAC_ADDR`).